### DISCUSSION TITLE
Customize AI-list

### DISCUSSION THREAD

**Discussion by WanderFox** [KindaBrazy/LynxHub#8] [Ideas]
It will be great to add feature that makes possible to add custom AI. For example: some users have multiple CimfyUI for different workflows (like 2d and 3d generation coz of different pytorch versions). And same for the Forge version of AUTOMATIC1111

**Comment by KindaBrazy** [comment]
Oh, I hadn’t considered that someone might want to have a WebUI multiple copies, like multiple ComfyUI directories.
 
I’m currently working on adding the forge to the module and definitely planning to work on enabling the ability to have the same WebUI installed multiple times.

Thank you for the suggestion!
